### PR TITLE
downgrade sample with permissions to 22

### PR DESCRIPTION
**Pull request by proppy**
```
android-ndk 🍊  grep -l uses-permission */app/src/main/AndroidManifest.xml
MoreTeapots/app/src/main/AndroidManifest.xml
Teapot/app/src/main/AndroidManifest.xml
audio-echo/app/src/main/AndroidManifest.xml
native-audio/app/src/main/AndroidManifest.xml
native-codec/app/src/main/AndroidManifest.xml
native-media/app/src/main/AndroidManifest.xml
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ before_install:
 script:
   - for f in */README.md; do sed -n '/Support/,/License/p' $f > /tmp/$(dirname $f).readme; done && diff -u --from-file=/tmp/hello-jni.readme /tmp/*.readme
   - test "$(grep -H compileSdkVersion */app/build.gradle | tee /dev/stderr | cut -d= -f 2 | sort | uniq | wc -l)" = "1"
-  - test "$(grep -H targetSdkVersion */app/build.gradle | tee /dev/stderr | cut -d= -f 2 | sort | uniq | wc -l)" = "1"
+  - test "$(grep -H targetSdkVersion */app/build.gradle | tee /dev/stderr | cut -d= -f 2 | sort | uniq | wc -l)" = "2"
```
Comment (claywilkinson) [inline .travis.yml]: what is this testing for? maybe a comment will help?

Comment (proppy) [inline .travis.yml]: done.

**Comment by claywilkinson** [comment]
LGTM

**Comment by ggfan** [comment]
LGTM

**Pull request merged**